src/runtime/cmd/kata-runtime/kata-check.go

```diff
@@ -538,3 +543,40 @@ func genericCheckKVMExtensions(extensions map[string]kvmExtension) (map[string]i
 
 	return results, nil
 }
+
+func checkVersions() error {
+	// Extract version of Crio
+	crioVerCmd := exec.Command("crio", "--version")
+	crioVerArr, err := crioVerCmd.Output()
+
+	if err != nil {
+		fmt.Printf("Unable to determine crio version %s\n", err)
```

Comment:
How about adding some message like `maybe you are not using CRI-O` in the output to let the user know that this is not a critical error?